fashion-example-query/README.md

```diff
@@ -5,81 +5,101 @@
 - [Run the Hello World example using query language](#run-the-hello-world-example-using-query-language)
   - [🗝️ Key Concepts](#️-key-concepts)
   - [🏃 Run the Flows](#-run-the-flows)
-  - [Results](#results)
-  - [QueryLanguage](#querylanguage)
+  - [🌟 Results](#results)
+  - [🧞‍QueryLanguage](#querylanguage)
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-# Run the Hello World example using query language
+# Run Fashion-MNIST Example Using Query Language
 
-## 🗝️ Key Concepts
+This is an extension of the Hello World example using QueryLanguage. 
+We will use the FashionMNIST dataset to index and query. The dataset has different categories such as t-shirts, trousers, and pullovers. We will use QueryLanguage to create a different index per each category so we can query through them. 
 
-First of all, read up on [Jina 101](http://101.jina.ai) so you have a clear understanding of how Jina works. We're going to refer to those concepts a lot. We assume you already have some knowledge of Python and machine learning.
 
-This is an extension of the Hello World example, but here we use QueryLanguage to create an index per category and then, with that we can filter by category
+## 🗝️ Expected knowledge 
 
+1. Read up on [Jina 101](http://101.jina.ai). 
+2. Python knowledge.
+3. [Hello World Example](https://docs.jina.ai/chapters/helloworld/index.html) understanding.
 
 
-### Install Requirements
+### Install requirements
 
-In your terminal:
+There are some requirements to run this example, to download everything you need, just run in your terminal:
 
 ```
 pip install -r requirements.txt
 ```
 
-
 ## 🏃 Run the Flows
 
-Now that we've got the code to load our data, we're going to dive into writing our app and running our Flows!
+We usually need at least two Flows in Jina. One for Indexing and one for Querying. 
 
 ### Index Flow
 
-To run the index you type:
-
-```bash
-python app.py -t index
-```
-First up we need to build up an index and then we'll search through this index when we use the query Flow later.
-So what is happening here?
+First thing we need is to index our data, and we need a Flow to manage this. 
 
-There are 10 categories in the fashion-mnist-data, but to simplify this example we will only use 3 categories:
+ ```python
+def index(num_doc, target: dict):
+    f = Flow.load_config('flows/index.yml')
+    with f:
+        f.index(index_generator(num_doc, target), request_size=2048)
+ ```
+      
+There are 10 categories in the fashion-mnist-data, but to simplify this example we will only use 3 of them:
 
     0	        T-shirt/top
     1	        Trouser
     2	        Pullover
-    
-We will create a separate index per category, so each category will have its own yml file and then we index the data updating updating the label
-
-```
+ 
+ We will create a separate index per category, so each category will have its own Yaml file. We can then index the data updating its label
```

Comment:
YAML

Reply:
I don't get the updating its label part?

Reply:
It's what the snippet code just below shows